frontend/src/components/AnnouncementsDisplay.tsx

```diff
@@ -0,0 +1,123 @@
+import { DialogButton, Focusable, PanelSection } from '@decky/ui';
+import { useEffect, useMemo, useState } from 'react';
+import { FaTimes } from 'react-icons/fa';
+
+import { Announcement, getAnnouncements } from '../store';
+import { useSetting } from '../utils/hooks/useSetting';
+
+const SEVERITIES = {
+  High: {
+    color: '#bb1414',
+    text: '#fff',
+  },
+  Medium: {
+    color: '#bbbb14',
+    text: '#fff',
+  },
+  Low: {
+    color: '#1488bb',
+    text: '#fff',
+  },
+};
+
+const welcomeAnnouncement: Announcement = {
+  id: 'welcomeAnnouncement',
+  title: 'Welcome to Decky!',
+  text: 'We hope you enjoy using Decky! If you have any questions or feedback, please let us know.',
```

Comment:
Where should people direct feedback?

(Also i assume this shows the first time you use decky? Might want to add something like start with looking at the store)

Reply:
That's a good point, but there is a blocking issue here. This strings need to be moves in en-US.json and not hardcoded here.